app/api/v1/endpoints/integrity.py

```
@@ -0,0 +1,349 @@
+async def get_merkle_proof_for_event(
+    event_seq: int,
+    tenant_id: Annotated[str, Depends(get_tenant_id)],
+    db=Depends(get_db),
+    _: Annotated[object, Depends(require_permission("tenant", "read"))] = None,
+):
+    """Return Merkle proof for a LEGAL_GRADE event identified by event_seq within its epoch."""
+    async for session in db:
+        event_repo = EventRepository(session)
+        epoch_repo = IntegrityEpochRepository(session)
+        merkle_repo = MerkleNodeRepository(session)
+        tsa_anchor_repo = TsaAnchorRepository(session)
```

Flake8 (7.3.0)

[error] local variable 'tsa_anchor_repo' is assigned to but never used

(F841)